eve/main.yml

```
@@ -2671,3 +2693,166 @@ stages:
+                )
+                [ "$VERSION" = "$LATEST_STABLE_RELEASE" ] && echo true || echo false
+            fi
+          env:
+            GIT_REFERENCE: '%(prop:git_reference)s'
+            IS_LATEST_DEV: '%(prop:is_latest_dev)s'
+            IS_LATEST_STABLE: '%(prop:is_latest_stable:-unset)s'
+            VERSION: '%(prop:metalk8s_version)s'
+      - SetProperty:
+          name: Set is_dev property
+          property: is_dev
+          value: false
+          doStepIf: '%(prop:use_production_registry:-false)s'
+      - ShellCommand:
+          name: Retrieve shell-ui image from pre-merge artifacts
+          command: >
+            curl -L -s -XGET -O '%(prop:premerge_artifacts_private_url)s/images/shell-ui.tar.gz'
+          haltOnFailure: true
+      - ShellCommand:
+          name: Load shell-ui image
+          command: docker load < shell-ui.tar.gz
+          haltOnFailure: true
+      - ShellCommand: &image_registry_tag_shell_ui
+          name: Tag shell-ui image with current version
+          command: |
+            if [ "$USE_PRODUCTION_REGISTRY" = true ]; then
+                PROJECT='%(secret:harbor_prod_project)s'
+            else
+                PROJECT='%(secret:harbor_dev_project)s'
+            fi
+            docker tag "$IMAGE_SRC" "$REGISTRY_HOST/$PROJECT/$IMAGE_DST"
+          env: &_env_image_registry_tag_shell_ui
+            <<: *_env_image_registry
+            IMAGE_SRC: 'shell-ui:v%(prop:metalk8s_version)s'
+            IMAGE_DST: 'shell-ui:v%(prop:metalk8s_version)s'
+      - ShellCommand:
+          <<: *image_registry_tag_shell_ui
+          name: Tag shell-ui image with commit short revision
+          env:
+            <<: *_env_image_registry_tag_shell_ui
+            IMAGE_DST: 'shell-ui:v%(prop:metalk8s_version)s-%(prop:commit_short_revision)s'
+          doStepIf: '%(prop:is_dev:-true)s'
+      - ShellCommand:
+          <<: *image_registry_tag_shell_ui
+          name: Tag shell-ui image as stable
+          env:
+            <<: *_env_image_registry_tag_shell_ui
+            IMAGE_DST: 'shell-ui:stable'
+          doStepIf: '%(prop:is_latest_stable)s'
+      - ShellCommand:
+          <<: *image_registry_tag_shell_ui
+          name: Tag shell-ui image as latest
+          env:
+            <<: *_env_image_registry_tag_shell_ui
+            IMAGE_DST: 'shell-ui:latest'
+          doStepIf: '%(prop:is_latest_dev)s'
+      - ShellCommand:
+          name: Push shell-ui image to the remote registry
+          command: |
+            if [ "$USE_PRODUCTION_REGISTRY" = true ]; then
+                PROJECT='%(secret:harbor_prod_project)s'
+            else
+                PROJECT='%(secret:harbor_dev_project)s'
+            fi
+            echo "The following $IMAGE_NAME tags will be pushed:"
+            docker images "$REGISTRY_HOST/$PROJECT/$IMAGE_NAME"
+            docker push "$REGISTRY_HOST/$PROJECT/$IMAGE_NAME"
```

Don't we want:
```suggestion
            docker push --all-tags "$REGISTRY_HOST/$PROJECT/$IMAGE_NAME"
```
here?

---

No, because this option does not exist in this docker version and it's the default behavior.